router/router.go

```diff
@@ -331,6 +282,90 @@ func (rt *HandleT) workerProcess(worker *workerT) {
 	}
 }
 
+func (rt *HandleT) handlePrevFailedUserJob(job *jobsdb.JobT, paramaters ParametersT, userID string, worker *workerT, previousFailedJobID int64) bool {
```

Comment:
ParametersT seems a bit generic. Shall we change this to JobParametersT? 

Not needed, but WDYT?